api/handlers_test.go

```diff
@@ -19,18 +19,18 @@ import (
 	"github.com/stretchr/testify/suite"
 )
 
-
 func testCfg() *config.Config {
 	diagnosticsBundleDir, err := ioutil.TempDir("", "diagnostic-bundle")
 	if err != nil {
 		log.Fatal(err)
 	}
 
 	return &config.Config{
-		FlagRole:                 "master",
-		FlagDiagnosticsBundleDir: diagnosticsBundleDir,
-		FlagPort:                 1050,
-		FlagMasterPort:           1050,
+		FlagRole:                         "master",
+		FlagDiagnosticsBundleDir:         diagnosticsBundleDir,
+		FlagPort:                         1050,
+		FlagMasterPort:                   1050,
+		FlagDiagnosticsJobTimeoutMinutes: 1,
```

Comment:
not directly related to the PR, but it'd be good to have it called `FlagDiagnosticsJobTimeout` and it'd take a `time.Duration`

Reply:
I agree but then it could change its semantic in `config.json` Ideally I'd like to create custom structure and handle it's serialization.